.openshiftio/Jenkinsfile.setup.groovy

```diff
@@ -0,0 +1,10 @@
+
+setupEnvironmentPre(env) {
+  sh "if ! oc get -n ${env} configmap app-config -o yaml | grep resource.configmap.yaml; then oc create -n ${env} configmap app-config --from-file=.openshiftio/resource.configmap.yaml; fi"
```

Comment:
I don't think `oc create configmap app-config --from-file=resource.configmap.yml` is what you want, because `resource.configmap.yml` is a Kubernetes resource (= intended to be used with `oc create` / `oc apply` etc.). What I think you really want is `oc create configmap app-config --from-file=src/main/etc/application.properties`.

Reply:
Ah yes, good catch, @Ladicek!

Reply:
You might need to specify the module too: `greeting-service/src/main/etc/application.properties`